modin/engines/ray/utils.py

```diff
@@ -15,6 +15,8 @@
 
 import os
 import sys
+import psutil
```

Comment:
I think this needs to be in requirements if we import it here.

Reply:
`psutil` package comes with `ray` package. Should we add explicitly dependency from `psutil` in our requirements files?

Reply:
@devin-petersohn what do you think?

Reply:
`psutil` is a hard dependency of Ray, I don't think we need to re-iterate that.